docs/components/OAuthProviderInstructions/content/components/StepTitle.tsx

```diff
@@ -1,13 +1,16 @@
-import { ReactNode } from "react";
+import { ReactNode } from "react"
+import styles from "../overrides.module.css"
 
 interface Props {
-  children: ReactNode;
+  children: ReactNode
 }
 
 export function StepTitle({ children }: Props) {
   return (
-    <h3 className="font-semibold tracking-tight text-slate-900 dark:text-slate-100 mt-8 text-2xl">
+    <h3
+      className={`font-semibold tracking-tight text-slate-900 dark:text-slate-100 mt-8 text-2xl ${styles["step-title-align"]}`}
+    >
```

Comment:
In the long run we should find a way to render these instructions from markdown. I think the challenge there is that neither MDX have a clear built-in concept of "dynamic markdown".

When building I explored [Next remote docs support](https://the-guild.dev/blog/nextra-3#remote-docs-support), which is kinda remote MDX, but didn't really cater for this use case (_where you have a markdown components and want to dynamic change its content according to the URL param which is selected provider_). Something to look up in the future 🙇🏽

Reply:
Yeah this use-case is tricky cause as you say there's no "dynamic markdown" really. Its fine for now I'd say :pray: